src/System.Net.Primitives/src/System/Net/CredentialCache.cs

```diff
@@ -12,27 +13,16 @@ namespace System.Net
     // name-password pairs and associates these with host/realm.
     public class CredentialCache : ICredentials, ICredentialsByHost, IEnumerable
     {
-        private readonly Dictionary<CredentialKey, NetworkCredential> _cache = new Dictionary<CredentialKey, NetworkCredential>();
-        private readonly Dictionary<CredentialHostKey, NetworkCredential> _cacheForHosts = new Dictionary<CredentialHostKey, NetworkCredential>();
-        internal int _version;
-
-        /// <devdoc>
-        ///  <para>
-        ///    Initializes a new instance of the <see cref='System.Net.CredentialCache'/> class.
-        ///  </para>
-        /// </devdoc>
+        private Dictionary<CredentialKey, NetworkCredential> _cache;
+        private Dictionary<CredentialHostKey, NetworkCredential> _cacheForHosts;
+        private int _version;
+
         public CredentialCache()
         {
         }
 
-        /// <devdoc>
```

Comment:
We previously decided to leave the devdoc in place.(Although I agree this isn't consistent, at least in this file.)